### DISCUSSION TITLE
Exterior/Physics/2/14/

### DISCUSSION THREAD

**Discussion by giscus** [SAVIA7582/Exterior#6] [General]
# Exterior/Physics/2/14/

https://savia7582.github.io/Exterior/Physics/2/14/

**Comment by WeberGuan18** [comment]
希望补充一下例三的图

**Comment by LH-0124** [comment]
例8螺绕环产生的磁场应该是NI，所以磁通应该是N平方

**Reply by LH-0124** [reply]
嘶这题算的是互感，是我看错了

**Comment by Monika-dokidoki** [comment]
例6第二问的顺逆时针和历年卷答案好像不一样
